src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -278,10 +278,24 @@ export default class DefaultBrowserBehavior implements BrowserBehavior, Extended
     return this.isAndroid();
   }
 
+  requiresDisablingH264Encoding(): boolean {
+    return (
+      (this.isIOSSafari() && this.version() === '15.1.0') ||
+      ((this.isIOSChrome() || this.isIOSFirefox()) && /( OS 15_1 )/i.test(navigator.userAgent))
+    );
+  }
+
   // These helpers should be kept private to encourage
   // feature detection instead of browser detection.
   private isIOSSafari(): boolean {
-    return this.browser.name === 'ios' || this.browser.name === 'ios-webview';
+    return (
+      this.browser.name === 'ios' ||
+      this.browser.name === 'ios-webview' ||
+      (this.browser.name === 'safari' &&
+        /( Mac )/i.test(navigator.userAgent) &&
+        !!navigator.maxTouchPoints &&
```

Comment:
nit: This check is not needed as `0 > 1` or `undefined > 1` will be `false` anyway.

Reply:
Oops :D